package/MDAnalysis/lib/formats/cython_util.pyx

```diff
@@ -71,7 +71,7 @@ cdef class ArrayWrapper:
         self.data_type = data_type
         self.ndim = ndim
 
-    def __array__(self):
+    def __array__(self, copy=None):
```

Comment:
here and elswhere we should probably also add `dtype=None` before `copy=None`, similar to the revisions added in my PR upstream at: https://github.com/scipy/scipy/pull/20172

It doesn't seem to matter for testing purposes at the moment, but adding the `dtype=None` is more formally correct now.